load_testing/common_flows/flow_sign_in.py

```diff
@@ -50,3 +49,94 @@ def do_sign_in(context):
     )
 
     return resp
+
+
+def do_sign_in_user_not_found(context):
+    num_users = get_env("NUM_USERS")
```

Comment:
do we need to cast this to an int or anything?

Reply:
It is `int()`-ified in `load_testing.common_flows.flow_helper.random_cred()`